express-notes-api/index.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+const arrayJson = [];
```

Comment:
This array should be created inside the `app.get('/api/notes, middleware)` because otherwise, when you request the notes over and over, it will get bigger and bigger.